### PR TITLE
Add new FBType OR_16 for bitwise boolean OR calculation.

### PR DESCRIPTION
**Pull request by franz-hoepfinger-4diac**
A new FBType named OR_16 has been added to the codebase. This FBType is designed to calculate bitwise boolean OR operations on 16 input variables and produce a single output variable. The XML structure of the FBType includes event inputs, event outputs, input variables, and output variables necessary for performing the OR operation effectively.

### CODE REVIEW

**Comment by diplfranzhoepfinger** [comment]
in some Cases a 16 Input OR is needed. here it is.

File: data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="OR_16" Comment="FB to calculate bitwise boolean OR (generic FB)">
+	<Identification Standard="61131-3" Classification="standard bitwise boolean function" Description="Copyright (c) 2014 Profactor GmbH&#10; &#10;This program and the accompanying materials are made&#10;available under the terms of the Eclipse Public License 2.0&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#10;&#10;SPDX-License-Identifier: EPL-2.0" >
```
Comment (azoitl) [inline data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt]: several things are wrong in the copy right header.

**Review by azoitl — CHANGES_REQUESTED** [review]

File: data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt (hunk 1 above)
Comment (franz-hoepfinger-4diac) [reply data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt]: oh. i will fix.

File: data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="OR_16" Comment="FB to calculate bitwise boolean OR (generic FB)">
+	<Identification Standard="61131-3" Classification="standard bitwise boolean function" Description="Copyright (c) 2014 Profactor GmbH &#10;  &#10;This program and the accompanying materials are made &#10;available under the terms of the Eclipse Public License 2.0 &#10;which is available at https://www.eclipse.org/legal/epl-2.0/ &#10; &#10;SPDX-License-Identifier: EPL-2.0" >
```
Comment (azoitl) [inline data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt]: I think I was not clear enough the (c) line should be:

Copyright (c)  2014, 2024 Profactor GmbH, HR Agrartechnik GmbH

As you took a file created by profactor and modified it. So both you and profactor have a copyright on that new file.

**Review by azoitl — CHANGES_REQUESTED** [review]

**Review by azoitl — APPROVED** [review]

**Pull request merged**